### DISCUSSION TITLE
how  switch between tabs by swiping left and right

### DISCUSSION THREAD

**Discussion by GodGavin** [cogentcore/core#1404] [Q&A]
hi，I want to enable users to switch between tabs by swiping left and right. Can you provide me with an example of how to implement this feature using ，thanks. And i also want enable user scroll in every single tab.

**Comment by kkoreilly** [comment]
@GodGavin Thank you for asking this. Please see my response here: https://github.com/cogentcore/core/issues/1399#issuecomment-2563931983.
